docs/Getting-Started.md

```diff
@@ -0,0 +1,178 @@
+        var Todos = this.state.todos.map(function(todo) {
+          return (
+            <Todo todo={todo} />
+          );
+        });
+
+        return (
+          <div>
+            <TodoCreateForm />
+            {Todos}
+          </div>
+        );
+      }
+    });
+```
+
+`Tux` also provides the ability to build high performance components out of the box. In our Todo application within the individual Todo component, a Todo's text is a `mutableTraits` property. Additionally the Mutable class is an Ownee component, so it allows you to validate the props that have been passed down to that individual component using `nearestOwnerPropTypes`.
+
+A high performance component:
+
+```javascript
+    var React = require('tux/React');
+
+    var Todo = React.createMutableClass({
+      mutableTraits: {
+        props: 'text'
+      },
+
+      propTypes: {
+        todo: React.PropTypes.object.isRequired
+      },
+
+      nearestOwnerPropTypes: {
+        remove: React.PropTypes.func.isRequired
+      },
+
+      handleRemove: function (e) {
+        e.preventDefault();
+        this.nearestOwnerProps.remove(this.props.todo);
+      },
+
+      render: function () {
+        return (
+          <p>{ this.props.todo.text }</p>
+          <button onClick={this.handleRemove}>Delete</buttion>
+        );
+      }
+    });
+
+    module.exports = Todo;
+```
+
+Another of the opinionated `Tux` components are the Ownee class. These components don't manage state, unless it is needed for input form validation. They don't need to manage their own state standardly as they receive their static methods and props from their closest Owner component. The `TodoCreateForm` utilizes this by accessing the `add` method from its nearest owner component, which is the `TodoViewOwner`.
+
+```javascript
+    var TodoCreateForm = React.createOwneeClass({
+      nearestOwnerPropTypes: {
+        add: React.PropTypes.func.isRequired
+      },
+
+      handleSubmit: function (e) {
+        e.preventDefault();
+        var todoTextInput = this.refs.textInput.getDOMNode();
+
+        this.nearestOwnerProps.add({
+          text: todoTextInput.value
+        });
+
+        todoTextInput.value = '';
+      },
+
+      render: function(){
+        return (
+          <form onSubmit={this.handleSubmit}>
+            <input type="text" ref="textInput" placeholder="Add a Todo" />
+            <button type="submit">Add</button>
+          </form>
+        );
+      }
+    });
+
+    module.exports = TodoCreateForm;
+```
+
+Tux abstracts away the complexity of Flux with powerful `Actions` syntax:
+
+```javascript
+    var Actions = require('tux/Actions');
+
+    var todoActions = Actions.createActionCategory({
+      category: 'todos',
+      source: 'todo views',
+      actions: ['add', 'remove']
+    });
+
+    module.exports = todoActions;
+```
+
+Tux provides all of the glue code needed to build stores and register them with the `TuxActions` dispatcher:
```

Comment:
Explain this more:

Now that we have created our actions and views. Let's create our store for our Todos that respond to our actions and provide the data for our views. `Tux` here again significantly reduces the boilerplate required to get a store up and running.